pkg/labels/labels.go

```diff
@@ -129,6 +129,22 @@ func (ls Labels) Hash() uint64 {
 	return xxhash.Sum64(b)
 }
 
+// HashWith returns a hash value for the two label sets.
+func (ls Labels) HashWith(w Labels) uint64 {
+	b := make([]byte, 0, 1024)
```

Comment:
@brian-brazil should this be increased to 2048?

Reply:
Why? Even large metrics rarely go over 200.